bioblend/galaxy/invocations/__init__.py

```diff
@@ -247,8 +249,36 @@ def get_invocation_biocompute_object(self, invocation_id):
         url = self._make_url(invocation_id) + '/biocompute'
         return self._get(url=url)
 
+    def wait_for_invocation(self, invocation, timeout_seconds=10):
```

Comment:
I think we have removed all calls to this method from tests <19.09, so I'd lean towards having `invocation_id` as argument here and call `self.gi.invocations.show_invocation(invocation_id)` below.
Given that all other methods in this module also supports only >=19.09 I think it's fair.